testing/run-scripts/run_cloud.sh

```diff
@@ -64,7 +65,17 @@ then
     fi
     BANNER=`docker exec uproxy-sshd cat /banner`
   else
-    BANNER=`hostname`
+    # DigitalOcean's metadata API can tell us the region in which
+    # the machine is located:
+    #   https://developers.digitalocean.com/documentation/metadata/#metadata-api-endpoints
+    BANNER=`curl -s -m 2 http://169.254.169.254/metadata/v1/region`
```

Comment:
That is exactly right. The `-m 2` means curl will time out after two seconds if the API is unavailable.